docs/module_web_with_python.md

```diff
@@ -5,137 +5,437 @@
 #### Explain the process of calculating the average value in a list of numbers!
+* We iterate over every object in the list, sum up their values, then divide them with the ammount of the numbers in the list.
 
 ### Procedural
 #### How the CASE condition works in SQL?
+* The CASE condition returns a value based on a set of conditions.
+* The CASE condition uses WHEN to declare responses based on the result of it's condition.
 #### How the switch-case condition works in JavaScript?
+* The switch statement checks a variable's value.
+* The case statement does something according to what the variables value is.
+* Essentially, the switch-case condition can act as a translation layer for data input.
 #### How to achieve a switch-case-like structure in Python?
+* Via the usage of dictionaries.
+* We declare a dictionary, then translate the input by referencing it on the dictionary's keys.
 #### Explain variable scoping in Python!
+* A variable created inside a function is only accessible in that function.
+* Variable Shadowing: When a local variable has the same name as that of an outer-scope variable. (Bad)
 #### What’s the difference between const and var in JavaScript?
+* Var elements' values can be altered after declaring them.
+* Const elements' values cannot be altered after declaring them.
 #### How the list comprehension looks like in Python?
+* fruits = ['apple', 'banana', 'cherry']
+* newlist = [x for x in fruits if 'a' in x] <--this returns a list with only those fruits that have the letter 'a' in them.
 #### How the “ternary expression” looks like in Python?
+* return 'a' if (c > b) else 'f' <-- This returns 'a' if c is bigger than b, else it return 'f'.
 #### How the ternary expression looks like in JavaScript?
+* return c > b ? 'a' : 'f' <-- This returns 'a' if c is bigger than b, else it return 'f'.
 #### How to import a function from another module in Python?
+* 'from [module] import [function] as [rename]' <-- Imports a single function from a module and renames it.
+* 'import [module]' <-- Imports all functions from a module.
+* 'from [module] import [function1], [function2]' <-- Import multiple functions from a single module.
+* 'import [module], [module]' <-- Import all functions from multiple modules
 #### How to import a function from another module in JavaScript?
+* import defaultExport from "module-name"; <-- imports the defaultExport from a module.
+* import { export1 } from "module-name"; <--Import a single export from a module.
+* import { export1 as alias1 } from "module-name"; <-- Import a single export from a module and rename it.
+* import { export1 , export2 } from "module-name"; <-- Import multiple exports from a single module.
 
 ### Functional
 #### What is recursion?
+* A recursive  function is when a function calls itself again based on a set of conditions.
 #### Write a recursive function which calculates the Fibonacci numbers!
+*	def fib(n):
```

Comment:
Forgot to format code for Python. Code is written in one line -> harder to read.

Reply:
?